make/istio/stop

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+: "${GIT_ROOT:=$(git rev-parse --show-toplevel)}"
+source "${GIT_ROOT}/make/include/has_namespace"
+source "${GIT_ROOT}/make/include/has_release"
+source "${GIT_ROOT}/make/include/defaults"
+
+printf "Stopping and cleaning up %s release...\\n" "${ISTIO_HELM_RELEASE}"
+
+if has_release "${ISTIO_HELM_RELEASE}"; then
+  helm delete --purge "${ISTIO_HELM_RELEASE}"
+fi
+
+if has_namespace "${ISTIO_NAMESPACE}"; then
+  kubectl delete namespace "${ISTIO_NAMESPACE}"
+fi
+
+# Delete Custom Resource Definitions.
+custom_resource_definitions=$(kubectl get customresourcedefinitions -o=go-template='{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}' | grep "istio.io" || true)
+echo "${custom_resource_definitions}" | xargs --no-run-if-empty kubectl delete customresourcedefinitions
```

Comment:
Not necessary, just thought it might be nice:
```bash
kubectl get customresourcedefinitions -o=go-template='{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}' \
  | grep "istio.io" \
  | xargs --no-run-if-empty kubectl delete customresourcedefinitions
```

Reply:
After the changes I made, I couldn't see it at the end. :)